Assets/Python/Consts.py

```diff
@@ -58,39 +50,34 @@
 
 # civilizations, not players
 iNumCivilizations = 57
-(iCivAmerica, iCivArabia, iCivArgentina, iCivAztec, iCivBabylonia, iCivBrazil, iCivByzantium, iCivCanada, iCivCarthage, iCivCelt, 

```

Comment:
Also, Mongols is preferred over Mongolia so if anything all other identifiers should be changed.